react/tic-tac-toe/src/services/AuthService.js

```diff
@@ -0,0 +1,5 @@
+import api from '~config/api';
+
+export default {
+  login: user => api.post('/login', { ...user })
```

Comment:
You don't need to spread the user object here since you're not changing anything from it:
```suggestion
  login: user => api.post('/login', user)
```